.github/ISSUE_TEMPLATE/gh-profile-review.md

```diff
@@ -9,6 +9,6 @@ assignees: ''
 **Github Profile Link:**
 <!-- your github profile link goes here. -->
 
-You can also join the discord community [here](https://discord.com/invite/jZQs6Wu)
+You can also join the discord community [here](http://discord.eddiehub.org)
```

Comment:
I think it should be https://discord.eddiehub.org

Reply:
No it's `http:` so those changes are good :) see [here](https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/issues/163#issuecomment-826338069)

Reply:
Oh, thanks for letting me know. Actually, when I pasted it in the browser it automatically turned into `https:` so I thought it was to be `https:`